processing/src/main/java/org/apache/druid/math/expr/BuiltInExprMacros.java

```diff
@@ -25,20 +25,28 @@
 import javax.annotation.Nullable;
 import java.util.Collections;
 import java.util.List;
+import java.util.Optional;
 import java.util.stream.Collectors;
 
 public class BuiltInExprMacros
 {
   public static class ComplexDecodeBase64ExprMacro implements ExprMacroTable.ExprMacro
   {
     public static final String NAME = "complex_decode_base64";
+    public static final String ALIAS_NAME = "decode_base64_complex";
```

Comment:
If we change this alias in the future is there a way to maintain backward compatibility?

Also, we might want to add native tests with the alias name to ensure the correctness

Reply:
we can not change the alias name once defined else it will regression, we need to add more alias. 
cool will add native test.